Sources/Plasma/Apps/plClient/CMakeLists.txt

```diff
@@ -109,28 +114,35 @@ elseif(APPLE)
         Mac-Cocoa/PLSPatcher.h
         Mac-Cocoa/PLSServerStatus.h
     )
-    set(RESOURCES 
+    list(APPEND plClient_RESOURCES
+        Mac-Cocoa/Assets.xcassets
+        Mac-Cocoa/banner.png
+        Mac-Cocoa/banner@2x.png
+    )
+    set(plClient_IBSOURCES
         Mac-Cocoa/MainMenu.xib
         Mac-Cocoa/PLSLoginWindowController.xib
         Mac-Cocoa/PLSPatcherWindowController.xib
-        Mac-Cocoa/banner.png
-        Mac-Cocoa/banner@2x.png
-        win32/res/Dirt.ICO
     )
-    
-    list(APPEND plClient_RESOURCES
-        win32/res/Dirt.ICO
+    list(APPEND plClient_EXTRA_RESOURCES
+        ${plClient_IBSOURCES}
     )
 else()
     list(APPEND plClient_SOURCES
         main.cpp
     )
 endif()
 
+plasma_executable(plClient CLIENT INSTALL_PDB
+    SOURCES
+        ${plClient_SOURCES} ${plClient_HEADERS}
+        ${plClient_TEXT} ${plClient_RESOURCES}
+)
+
 if(APPLE)
-    plasma_executable(plClient CLIENT INSTALL_PDB SOURCES ${plClient_SOURCES} ${plClient_HEADERS} ${RESOURCES})
+    target_xib_resources(plClient SOURCES ${plClient_IBSOURCES})
```

Comment:
Alternatively, you could process the XIBs in the `plasma_executable` command, which would be more ideal in terms of limiting the platform-specific branches in consuming code.

Reply:
you mean adding another keyword like `plasma_exectuable(plClient CLIENT INSTALL_PDB SOURCES ... IBSOURCES ...)`? Or trying to parse out `.xib` files from the list of provided sources?

Reply:
I was thinking more of the latter.